src/parsing/av3C.js

```diff
@@ -0,0 +1,546 @@
+/*
+ * Copyright (c) 2023. Paul Higgs
+ * License: BSD-3-Clause (see LICENSE file)
+ */
+
+var OSisLittleEndian=false;
```

Comment:
Global variables should be avoided. 
Also, why would AVS3 need endianness considerations compared to other codecs?

Reply:
The bit reader can return values either as big endian of little endian (useful with native code).
I can fake this out on the assumption  that all will be little endian, of just put it in the class.